serbian/alkohol/alkohol.md

```diff
@@ -0,0 +1,2 @@
+Vekovima optuzivana supstanca za drustveno neprihvatljivo ponasanje. To jutro kao i prethodno posle [paba](https://en.wikipedia.org/wiki/Pub) svratio je u pekaru, nasmejao se i [upitao je nesto](../burekSir/noz/noz.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.